docs/proposed/config-split.md

```diff
@@ -90,7 +93,7 @@ Open question: do we group them into a subsection?
 
 ## Repartition of the configuration options
 
-### In tf_psa_crypto_config.h, we have:
+### In `tf_psa_crypto_config.h`, we have:
 * SECTION "Platform abstraction layer"
```

Comment:
This will render to something unreadable: https://github.com/ronald-cron-arm/mbedtls/blob/config-file-split/docs/proposed/config-split.md#in-tf_psa_crypto_configh-we-have Suggestion:

    #### SECTION ...
    ```
    # define ...
    ```

Reply:
indeed, thanks.